core/web_admin.py

```
@@ -0,0 +1,1040 @@
+            if auth_header.lower().startswith("bearer "):
+                return auth_header[7:].strip()
+            return (
+                request.headers.get("X-Admin-Token", "").strip()
+                or request.args.get("token", "").strip()
+                or (request.get_json(silent=True) or {}).get("token", "")
+            )
+
+        def require_auth(handler):
+            def wrapped(*args, **kwargs):
+                if extract_token() != self.token:
+                    return json_error("未授权：请提供正确的 Web Token。", 401)
```

Ruff (0.15.15)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)